cloudflare.install.sh

```diff
@@ -208,11 +208,15 @@ install $SOURCE_DIR/APIKey $INSTALL_DIR/bin/admin/CloudFlare/APIKey
 chmod 0700 $INSTALL_DIR/bin/admin/CloudFlare/APIKey
 echo "mode=simple" > $INSTALL_DIR/bin/admin/CloudFlare/APIKey.conf
 
+# Get PHP Version
+CPANELSUPPORTEDPHPPATH=`ls -al $INSTALL_DIR/3rdparty/bin/php | awk '{print $11}'`
+PHPVERSION=`echo $CPANELSUPPORTEDPHPPATH | rev | cut -d '/' -f 3 | rev`
```

Comment:
What about 
```
#!/bin/bash
IN="/usr/local/cpanel/3rdparty/php/56/bin/php"
IFS="/" && Array=($IN)
PHPVERSION=${Array[6]}
echo $PHPVERSION
```
http://stackoverflow.com/questions/918886/how-do-i-split-a-string-on-a-delimiter-in-bash

This rev, cut, rev logic scares me because I don't know bash and if they change the path its not clear how to fix.